src/main/java/com/uid2/shared/auth/Roles.java

```diff
@@ -10,7 +10,7 @@
 
 public class Roles {
     public static <T extends Enum<T>> Set<T> getRoles(Class<T> type, JsonObject clientKeyJson) {
-        return getRoles(type, clientKeyJson.getJsonArray("roles"));
+        return getRoles(type, clientKeyJson.getJsonArray("roles", new JsonArray()));
```

Comment:
Would it be better to pass the "roles" field directly into this rather than the whole `JsonObject`?

Reply:
There are two `getRoles`, the second one we are calling from here is just taken the roles field, this is the wrapper, should we remove this wrapper but calling `getRoles(type, clientKeyJson.getJsonArray("roles", new JsonArray()));` from OperatorKey and ClientKey?